pquery-run.sh

```diff
@@ -1408,6 +1422,9 @@ pquery_test(){
         if [ $X -ge ${PQUERY_RUN_TIMEOUT} ]; then
           echoit "${PQUERY_RUN_TIMEOUT}s timeout reached. Terminating this trial..."
           TIMEOUT_REACHED=1
+          echoit "TIMEOUT_INCREMENT option was enabled and set to ${TIMEOUT_INCREMENT} sec"
+          echoit "${TIMEOUT_INCREMENT}s will be added to the next trial timeout."
```

Comment:
can we move this and show in the starting, running the test for N seconds. 
would be more relevant.

Reply:
It will be seen in the beginning of every trial. Reason why it is put at the end of the iteration is, we want the first trial to run without increment which was happening earlier